README.md

```diff
@@ -103,6 +103,25 @@ Currently the Cld2 bindings are only generated for `linux-x86-64`, so if your
 machine is different it probably won't work. In such a case, just execute it
 with the `java_only` profile.
 
+Launch service using docker
+--------------------------
+The service can be also launched using docker. It requires the installation of `docker` and `docker-compose`. See [docker home page](https://docs.docker.com/) for installation references.
+
+Once docker is installed execute
+```bash
+$ docker-compose up -d
+```
+this will build the image of the service and launch it. To see on which port the service is exposed,
```

Comment:
Remove this part, as I prefer to expose the port 8080 to port 8080. Instead, what needs to be added is that in order to call the service one needs to use the ip of the VM. Which can be obtained with `boot2docker ip`.